snow/validators/validatorstest/state.go

```diff
@@ -80,3 +82,16 @@ func (vm *State) GetValidatorSet(
 	}
 	return nil, errGetValidatorSet
 }
+
+func (vm *State) GetCurrentValidatorSet(
+	ctx context.Context,
+	subnetID ids.ID,
+) (map[ids.ID]*validators.GetCurrentValidatorOutput, uint64, bool, error) {
+	if vm.GetCurrentValidatorSetF != nil {
+		return vm.GetCurrentValidatorSetF(ctx, subnetID)
+	}
+	if vm.CantGetCurrentValidatorSet && vm.T != nil {
+		require.FailNow(vm.T, "unexpectedly called GetCurrentValidatorSet")
+	}
+	return nil, 0, false, nil
```

Comment:
nit: I suppose the pattern would be to make a new `errGetCurrentValidatorSet` as the final return value here.
not sure it matters except when vm.T is not set